client/src/app.jsx

```diff
@@ -1,20 +1,78 @@
-import React, { Component } from 'react';
-import './app.css';
+import React from 'react';
+import axios from 'axios';
+// import './app.css';
+
+import ContainerAverageStars from './components/ContainerAverageStars.jsx';
+import ContainerReviews from './components/ContainerReviews.jsx';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
 
-    this.state({
-      example: 'example',
-    });
+    this.getListingAverageStars = this.getListingAverageStars.bind(this);
+    this.getListingReviews = this.getListingReviews.bind(this);
+
+    // Just for testing
+    this.logState = this.logState.bind(this);
+
+    this.state = {
+      averageStarsObj: {},
+      reviews: [],
+    };
+  }
+
+  componentDidMount() {
+    // Using parameter 1 for now, just for testing
+    this.getListingReviews(1);
+    this.getListingAverageStars(1);
+  }
+
+  getListingAverageStars(id) {
+    axios.get(`/api/listings/${id}/averagestars`)
+      .then((response) => {
+        // console.log(response.data);
+        this.setState({
+          averageStarsObj: response.data,
+        });
+      })
+      .catch((error) => {
+        // Not finished! Will properly error handle in the future.
+
+        // console.log(error);
+      });
+  }
+
+  getListingReviews(id) {
+    axios.get(`/api/listings/${id}/reviews`)
+      .then((response) => {
+        // console.log(response.data);
+        this.setState({
+          reviews: response.data,
+        });
+      })
+      .catch((error) => {
```

Comment:
Good job properly handling errors. It's common to forget.

Reply:
Thanks Justin.  Haven't quite handled them "properly" yet, but that's the goal!